panel/src/components/Layout/Table.vue

```diff
@@ -398,7 +398,7 @@ export default {
 	color: var(--table-color-th-text);
 	background: var(--table-color-th-back);
 }
-.k-table th[data-has-button] {
```

Comment:
While going through all the examples, I ran into a breaking issue in the lab. If you ever use one of the attributes manually and you don't make it strict, this is basically a breaking change. I wonder how we deal with this.

Reply:
What do you mean with using them manually?

Reply:
In the lab, we use `<th data-has-button>` in our example markup. That used to work so far, but does no longer work now.

Reply:
Hmmm we could for the moment to add selectors for both. But those use cases will break sooner or later anyway when Vue renders `data-attribute="false"` ad those selectors just say "should exist at all" not whether they are true/positive.

Reply:
That's true. I just really wonder if we need to consider this a breaking change or not. Feels still a bit new to get used to such cases.

Reply:
We could also add it to `v5/develop` instead and list it as breaking change.

Reply:
I think if we want to be truly carefuly, we should do it like that. It feels at least weird to do that already in 4.1, that's why I'm leaning towards that suggestion tbh.